vtex-apps/login
# Added step CSS Handle to LoginContent Component

#### What is the purpose of this pull request?

Added step CSS Handle to LoginContent Component

#### What problem is this solving?

Style each step accordingly

#### How should this be manually tested?
With the pull-request updates, inspect the login content div and see the step handle changing accordingly.

#### Screenshots or example usage
http://thirdbranch--storeido.myvtex.com/
![image](https://user-images.githubusercontent.com/7662734/82829239-2d8aca00-9e89-11ea-80f6-fdae960108c9.png)


#### Types of changes
